package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aemm-plugin-context",
-  "version": "1.1.2-dev",
+  "version": "1.2-dev",
```

Comment:
Thanks, this helps me to know that the next increment will be a minor version instead of a patch.